astronomer-starship-provider/README.rst

```diff
@@ -46,28 +40,47 @@ Usage
           task_id='export_meta',
           deployment_url='{{ dag_run.conf["deployment_url"] }}',
           token='{{ dag_run.conf["astro_token"] }}',
+          variables_exclude_list=[""],
+          connection_exclude_list=[""],
+          env_include_list=[""]
```

Comment:
Let's leave these off since they are specified in the `Configuration` section.

Reply:
@pohek321 I thought it was kinda confusing to not have them in the DAG but then add extra params in the config sections - then I added more info to the config section to give clear use instructions. Thoughts? Can remove if you disagree

Reply:
@pohek321 what version should it be bumped to in setup.cfg?

Reply:
@fritz-astronomer and I talked it over as well. We landed on just having them in the `Configuration` section. I also don't know what will happen if you specify an empty list on the param. I know if the param isn't specified or is `None` it works.